src/app/components/stopwatch/example/terra-stopwatch.component.example.ts

```diff
@@ -0,0 +1,20 @@
+import {
+    Component,
+    ViewChild
+} from '@angular/core';
+import { TerraStopwatchComponent } from '../terra-stopwatch.component';
+
+@Component({
+    selector: 'terra-stopwatch-example',
+    styles:   [require('./terra-stopwatch.component.example.scss')],
+    template: require('./terra-stopwatch.component.example.html'),
+})
+export class TerraStopwatchComponentExample
+{
+    @ViewChild('viewChildCurrentTime') public viewChildCurrentTime:TerraStopwatchComponent;
```

Comment:
`public viewChildCurrentTime:TerraStopwatchComponent;` Muss in die nächste Zeile